custom_components/llama_conversation/__init__.py

```diff
@@ -302,7 +309,11 @@ async def async_process(
             )
 
         conversation.append({"role": "assistant", "message": response})
-        self.history[conversation_id] = conversation
+        if remember_conversation:
+            if remember_num_interactions and len(conversation) > remember_num_interactions * 2:
```

Comment:
shouldn't this be `len(conversation) > (remember_num_interactions * 2) + 1` because of the system prompt at the start?

Reply:
Yes it should, this is what I get for coding at 6 am, thanks for reviewing!